tools/bridge/bridge/validation_utils.py

```diff
@@ -0,0 +1,73 @@
+from web3 import Web3
```

Comment:
Yes, the naming was a big problem for me here. I felt forced to change name, because I consider the account verification to not be a contract validation.

> I expected all config validation functions in here,

I understand this. The problem is that we need a two layered validation of the configuration. First is based on each option standalone and focus in first place on the type and format. I would call this a syntactically check. Moreover this layer makes sure that all necessary options are defined and default values are applied.
The second layer is for the semantic check to make sure the configured options are what they claim to be. This includes the interoperability of the configuration values. So must I use the foreign chain RPC address to verify the token contract for example. This is not possible within layer one.

That's my view to the things. Help me finding a satisfying name.

Reply:
Some suggestions without a clear preference from my side:

- `contract_validation.py`
- `chain_validation.py`
- `rpc_validation.py`

Reply:
Decided for `contract_validation`. I'm not 100% satisfied, but I fail to suggest anything better.

Reply:
:heavy_check_mark: